### ISSUE TITLE
Errata01 20230619 Section 2.4 Syntax notation - Second list item - ECMA ref link broken

### ISSUE THREAD

**Issue by sthagen** [oasis-tcs/sarif-spec#592]
When accepting all changes the link with the text ECMA404 in the second list item of section 2.4 Syntax notation is broken

> * The following “special sequence” (see EBNF [ISO14977:1996], §4.19 and §5.11) refers to any character that can appear in a JSON string according to JSON >>>[ECMA404]<<<:

Could be an application specific problem.

Proposal: If possible make them work.

Reference bundle <https://www.oasis-open.org/committees/document.php?document_id=71131&wg_abbrev=sarif> and file `sarif-v2.1.0-errata01-csd01-redlined.docx` (section 2.4 page 25 (on my machine) second list item)

PS: I thought we use "Title Case for Section Titles" - but I am not a native English reader.

**Comment by dmk42** [comment]
Thanks!  Regarding ECMA404, the hyperlink existed and was valid, but we had never added the normative reference to which it was supposed to point.  This was true even in the original standard.  I've added the normative reference and am updating #568 to incorporate that fix.

Regarding title case, I think you are correct, but the way titles appear is consistent throughout the document except for the early parts.  I decided not to change this for now, but we can revisit this for SARIF v2.2.
